docs/sources/alerting/set-up/configure-high-availability/_index.md

```diff
@@ -55,7 +55,7 @@ ports:
     protocol: TCP
 ```
 
-2. Add the environment variables to the Grafana deployment:
+2. Add the environment variables to the Grafana deployment.
```

Comment:
I think this should be a style discussion because the `:` indicates that there is more to follow.